### ISSUE TITLE
Filter option is missing for the following tabs in the UI 

### ISSUE THREAD

**Issue by kiranchavala** [apache/cloudstack#7218]
##### ISSUE TYPE

 * Enhancement Request

##### COMPONENT NAME

~~~
Componet:UI

Componet:api
~~~

##### CLOUDSTACK VERSION

~~~
4.18 nightly
4.17.2

~~~

##### SUMMARY

The Filter option is missing for the following tabs in the UI 

This is a minor ui enhancement request but good to have for the admin users for quick search

~~~

1. Filter missing for listing Kubernetes cluster 

Compute > Kubernetes

https://cloudstack.apache.org/api/apidocs-4.17/apis/listKubernetesClusters.html


2. Filter is missing for AutoScale VM Groups

Compute >  AutoScale VM Groups

https://cloudstack.apache.org/api/apidocs-4.17/apis/listAutoScaleVmGroups.html

3. Filter is missing for Instance Groups

Compute > Instance groups

https://cloudstack.apache.org/api/apidocs-4.17/apis/listInstanceGroups.html

5. Filter is missing for SSH Key pairs

Compute > SSH key pairs

https://cloudstack.apache.org/api/apidocs-4.17/apis/listSSHKeyPairs.html

6. Filter is missing for Affinity groups

Compute > Affinity groups

https://cloudstack.apache.org/api/apidocs-4.17/apis/listAffinityGroups.html


7. Filter is missing for Public Ip Address

Network> Public Ip addresses

https://cloudstack.apache.org/api/apidocs-4.17/apis/listPublicIpAddresses.html

8. Kubernetes ISO filter option is missing from the UI 

Images > Kubernetes iso


https://cloudstack.apache.org/api/apidocs-4.17/apis/listKubernetesSupportedVersions.html


9. Filter is missing for Roles

Roles >

https://cloudstack.apache.org/api/apidocs-4.17/apis/listRoles.html

10. Filter is missing for Accounts

Accounts

https://cloudstack.apache.org/api/apidocs-4.17/apis/listAccounts.html

11. Filter is missing for zones

Infrastructure >Zones

https://cloudstack.apache.org/api/apidocs-4.17/apis/listZones.html


12. Filter is missing for pods

Infrastructure > Pods

https://cloudstack.apache.org/api/apidocs-4.17/apis/listPods.html

13. Filter is missing for cluster

Infrastructure > Cluster

https://cloudstack.apache.org/api/apidocs-4.17/apis/listClusters.html

14. Filter is missing for hosts

Infrastructure > hosts

https://cloudstack.apache.org/api/apidocs-4.17/apis/listHosts.html

15. Filter is missing for primary storage

Infrastructure > primary storage

https://cloudstack.apache.org/api/apidocs-4.17/apis/listStoragePools.html

16. Filter is missing for secondary storage

Infrastructure > secondary storage

https://cloudstack.apache.org/api/apidocs-4.17/apis/listImageStores.html

17. Filter is missing for system vm's

Infrastructure > system vm's

https://cloudstack.apache.org/api/apidocs-4.17/apis/listSystemVms.html


18. Filter is missing for Management server vm's

Infrastructure > Mangement server vm

https://cloudstack.apache.org/api/apidocs-4.17/apis/listManagementServers.html


19. Filter is missing for alerts

Infrastructure>Alerts

https://cloudstack.apache.org/api/apidocs-4.17/apis/listAlerts.html


Logged #7193

https://github.com/apache/cloudstack/issues/7193



20. Filter is missing for service offerings

Service offerings > Compute offerings

https://cloudstack.apache.org/api/apidocs-4.17/apis/listServiceOfferings.html


21.Service offerings > system offerings

https://cloudstack.apache.org/api/apidocs-4.17/apis/listServiceOfferings.html

22. Service offerings > Disk offerings

https://cloudstack.apache.org/api/apidocs-4.17/apis/listDiskOfferings.html

23. Service offerings > Network offerings

https://cloudstack.apache.org/api/apidocs-4.17/apis/listNetworkOfferings.html

24. Service offerings > vpc offerings

https://cloudstack.apache.org/api/apidocs-4.17/apis/listVPCOfferings.html

25. Configuration > LDAP configuration

https://cloudstack.apache.org/api/apidocs-4.17/apis/listLdapConfigurations.html


26. Configuration > Hypervisor Capablities 

https://cloudstack.apache.org/api/apidocs-4.17/apis/listHypervisorCapabilities.html

~~~

**Comment by kiranchavala** [comment]
@borisstoyanov  could you please add the labels

Componet:UI

Componet:api

type:enhancement

type:improvement

**Comment by shwstppr** [comment]
This will need specifying what filter parameters are needed for the different entities

**Comment by DaanHoogland** [comment]
should be fixed in #9068
